warehouse/models/mart/gtfs/dim_calendar_dates.sql

```diff
@@ -6,17 +6,23 @@ WITH make_dim AS (
 ),
 
 dim_calendar_dates AS (
-    -- some full duplicate rows
-    SELECT DISTINCT
-        {{ dbt_utils.generate_surrogate_key(['feed_key', 'service_id', 'date']) }} AS key,
+    SELECT
+        {{ dbt_utils.generate_surrogate_key(['feed_key', '_line_number']) }} AS key,
+        {{ dbt_utils.generate_surrogate_key(['feed_key', 'service_id', 'date']) }} AS _gtfs_key,
         feed_key,
         service_id,
         date,
         exception_type,
         base64_url,
+        _dt,
         _feed_valid_from,
+        _line_number,
         feed_timezone,
     FROM make_dim
+    -- filter rather than flag; lots of downstream models expect uniqueness
+    -- we could remove if we handled it everywhere but these are mostly full duplicates
+    -- and they occur on many feeds
+    QUALIFY ROW_NUMBER() OVER (PARTITION BY feed_key, service_id, date ORDER BY _line_number) = 1
```

Comment:
should have exception type too; if we have dups with conflicting exception types we have to figure something else out